src/overlays/effects/ovl_Effect_Ss_Extra/z_eff_ss_extra.c

```diff
@@ -1,29 +1,94 @@
 /*
  * File: z_eff_ss_extra.c
  * Overlay: ovl_Effect_Ss_Extra
- * Description:
+ * Description: The floating points hovering over a target you hit in OOT
+ *   IE: 30/60/100 points when hitting the target in lost woods with slingshot to get bigger seed sack
  */
 
 #include "z_eff_ss_extra.h"
+#include "objects/object_yabusame_point/object_yabusame_point.h"
 
 #define PARAMS ((EffectSsExtraInitParams*)initParamsx)
```

Comment:
What's going on with this macro? Is this something unique to effects?

Reply:
Yeah, this is basically the effects equivalent to the `THIS` macro. However, it only appears in the `Init` function, since it applies to the different `initParams` structs.